docs/creating_integration_file_structure.md

```diff
@@ -33,5 +33,5 @@ Home Assistant will look for an integration when it sees the domain referenced i
 - `<config directory>/custom_components/<domain>`
 - `homeassistant/components/<domain>` (built-in integrations)
 
-You can override a built-in integration by having an integration with the same domain in your `<config directory>/custom_components` folder. [The `manifest.json` file requires a version tag when you override a core integration](creating_integration_manifest/#version). An overridden core integration can be identified by a specific icon in the upper right corner of the integration box in the overview [![Open your Home Assistant instance and show your integrations.](https://my.home-assistant.io/badges/integrations.svg)](https://my.home-assistant.io/redirect/integrations/)
+You can override a built-in integration by having an integration with the same domain in your `<config directory>/custom_components` folder. [The `manifest.json` file requires a version tag when you override a core integration](creating_integration_manifest.md/#version). An overridden core integration can be identified by a specific icon in the upper right corner of the integration box in the overview [![Open your Home Assistant instance and show your integrations.](https://my.home-assistant.io/badges/integrations.svg)](https://my.home-assistant.io/redirect/integrations/)
```

Comment:
This is incorrect. What's not working for you?

Reply:
There is an invalid link on the page [Integration File Structure | Home Assistant Developer Docs](https://developers.home-assistant.io/docs/creating_integration_file_structure/). The link is near the bottom of the page and is named **The manifest.json file requires a version tag when you override a core integration**. It points to https://developers.home-assistant.io/docs/creating_integration_file_structure/creating_integration_manifest/#version which returns **Page Not Found**.

Reply:
This is actually a correct fix. Docusaurus will transform this into a correct link.

Reply:
Reverted, as it indeed breaks CI now in the latest version